eu.skribisto.skribisto.yml

```diff
@@ -0,0 +1,58 @@
+app-id: eu.skribisto.skribisto
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: skribisto
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+separate-locales: false
+modules:
+
+  - name: openjpeg2
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_INSTALL_LIBDIR=lib
+    sources:
+      - type: archive
+        url: https://github.com/uclouvain/openjpeg/archive/v2.3.0.tar.gz
+        sha256: 3dc787c1bb6023ba846c2a0d9b1f6e179f1cd255172bde9eb75b01f1e6c7d71a
+        
+  - name: sqlite3
```

Comment:
I assume here that the sqlite3 in the runtime isn't enough

Reply:
You are right, I don't need it with KDE runtime. I had need for it when I tried to use the freedesktop runtime. Same with openjpeg. I built it and ran it fine without these two steps.